lib/packmap.js

```diff
@@ -0,0 +1,82 @@
+const fs = require("fs-extra");
+const path = require("path");
+const sortObject = require("sort-object-keys");
+
+module.exports = async function packmap(opts) {
+  const normalizedOutdir = path.normalize(opts.outdir);
+  await fs.remove(normalizedOutdir);
+  await fs.ensureDir(normalizedOutdir);
+  const packageJsonPath = path.resolve(process.cwd(), opts.package);
+  const packageJson = require(packageJsonPath);
+  const importMap = await traversePackage(packageJson);
+  importMap.imports = sortObject(importMap.imports);
+  await fs.appendFile(
+    path.resolve(opts.outdir, "import-map.json"),
+    JSON.stringify(importMap, null, 2)
+  );
+
+  function traversePackage(packageJson, importMap = { imports: {} }) {
+    const dependencies = Object.keys(packageJson.dependencies || {});
+    return Promise.all(
+      dependencies.map(dependency => {
+        if (dependency.startsWith("@types/")) {
```

Comment:
I wonder if there are other packages you'll want to be able to skip. Or at least maybe add some configuration at some point to skip some deps

Reply:
Yeah I imagine this will become configurable over time